assets/components/paymentButtons/payPalContributionButton/payPalContributionButtonReducer.js

```diff
@@ -0,0 +1,52 @@
+// @flow
```

Comment:
Is it worth to add some sort of documentation, explaining that if a developer would like to use the PayPalContributionButotn, she or he should create a PayPal state in the pageReducer?

Reply:
Maybe it could live inside https://github.com/guardian/support-frontend/tree/master/assets/components/paymentButtons folder.

Reply:
You don't actually have to. The component itself is presentational regardless of what you happen to be doing with the state in a specific instance.

This button is a good example actually. In the support landing and UK contributions landing pages the reducer and corresponding actions are used. However, in other pages such as the US landing page, this error state isn't actually used, there's just a simple error handler and no concept of the error state being reset. The questions is whether we want to push developers into a particular way of using these components.

Reply:
As discussed offline, it would be nice to explain that somewhere in the codebase and since now we have two `containerisedComponents` I would create a folder to contain them. 

```
assets
    |- components
    |- containerisedComponents
    |    |-readme.md //Readme explaining how this kind of state works.
```

Reply:
The folder was created in this pr: https://github.com/guardian/support-frontend/pull/508